### DISCUSSION TITLE
Question about PF and IPF

### DISCUSSION THREAD

**Discussion by DoctorChenb** [mtex-toolbox/mtex#1320] [Ask Anything]
Hello everyone!

Here are the PF and IPF I plotted with Mtex code. 

![image](https://user-images.githubusercontent.com/97789085/150304045-24019294-813f-4782-b729-a001a2a0609a.png)

This result is the same as the Channel 5 , so the code is right. 
However, the PF shows basal texture,but the IPF does not,why?

Thank you very much in advance!

Kind Regards,
Chen bo

**Comment by kilir** [comment]
Hi Chen,
I would think that a coincidence of results with Channel 5 can not necessarily be taken as a confirmation that any of both is actually correct. Anyhow, please post the relevant lines of your script.

Cheers,
Rüdiger

**Reply by DoctorChenb** [reply]
Hi Rüdiger，
I feel honored to receive your reply!  My code is as follows.
```
plotPDF(odf,h(1),'projection','eangle','points',PointNum,'contourf',...
    'colorrange','tight','grid','grid_res',15*degree,'silent')  %'antipodal','upper',

plotIPDF(odf,h2(1),'projection','eangle','points',PointNum,'contourf',...
    'colorrange','tight','grid','grid_res',15*degree,'silent') % 'antipodal','upper',
```

Kind Regards,
Chen bo

**Reply by kilir** [reply]
Hi,
could you also supply the lines where you define h2?
Cheers,
Rüdiger

**Reply by DoctorChenb** [reply]
Hi ,
OK,thank you!
```
h=Miller({0,0,0,1},{1,0,-1,0},{1,1,-2,0},'hkil',cs);
h2=[xvector,yvector,zvector]; % for IPF,ED,TD,ND
```

Kind Regards,
Chen bo

**Reply by kilir** [reply]
Hi,
so you are plotting an ipf with respect to the xvector. Assuming you have this pointing to the east or west for your polefigure plot. I'd say what you get is quite reasonable since your polefigure shows that you also have some grains with 0001 oiiting towards this direction. It may help you if you'd colorcode your map with an ipf colorcoding with respect to the xvector, since that seems to be the one your are interested in.

Cheers,
Rüdiger
